### PR TITLE
Update NodeJS dependency to 18.x

### PR DESCRIPTION
**Pull request by ocket8888**
For all things that depend on NodeJS, this PR updates them to depend on/document a dependency on/use version 18. There were several unused Traffic Portal Dockerfiles that had fallen well behind and aren't actually supported, so I removed those instead.

<hr/>

## Which Traffic Control components are affected by this PR?
- Documentation
- Traffic Portal
- Automation (GitHub Actions, Docker images)

## What is the best way to verify this PR?
Everything should continue to work as normal, so make sure existing tests pass.

## PR submission checklist
- [x] This PR uses existing tests
- [x] This PR has documentation
- [x] This PR has a CHANGELOG.md entry
- [x] This PR **DOES NOT FIX A SERIOUS SECURITY VULNERABILITY**

### CODE REVIEW

File: infrastructure/cdn-in-a-box/traffic_portal/Dockerfile [hunk 1]
```diff
@@ -41,7 +41,6 @@ ARG TO_HOST=$TO_HOST
 RUN dnf install -y epel-release && \
     dnf install -y \
       jq \
-      nodejs \
```
Comment (shamrickus) [inline infrastructure/cdn-in-a-box/traffic_portal/Dockerfile:44]: You need [this line](https://github.com/nodesource/distributions#using-ubuntu-5). The Enterprise Linux section doesn't include the install nodejs part, but if you look at the `setup_XX.x` scripts they tell you to install nodejs afterwards. This is why the CiaB is failing with unable to find `npm`.

Comment (ocket8888) [reply infrastructure/cdn-in-a-box/traffic_portal/Dockerfile:44]: I figured, but I wanted to find out for myself. Letting GHA try to do it is perhaps not the _fastest_ way to find out, but it's by far the _easiest_.

File: infrastructure/cdn-in-a-box/traffic_portal/Dockerfile [hunk 2]
```diff
@@ -41,15 +41,14 @@ ARG TO_HOST=$TO_HOST
...
+	  nodejs \
```
Comment (shamrickus) [inline infrastructure/cdn-in-a-box/traffic_portal/Dockerfile:44]: Inconsistent whitespace

**Review by shamrickus — APPROVED** [review]
LGTM

**Pull request merged**